recipe/meta.yaml

```diff
@@ -18,34 +18,47 @@ build:
     - {{ pin_subpackage('pastix', max_pin='x.x') }}
   skip: true  # [win]
 
+outputs:
+  - name: {{ name }}
+    requirements:
+      build:
+        - {{ stdlib('c') }}                 # [unix]
+        - {{ compiler('c') }}               # [unix]
+        - {{ compiler('cxx') }}             # [unix]
+        - {{ compiler('fortran') }}         # [unix]
+        - {{ stdlib('m2w64_c') }}           # [win]
+        - {{ compiler('m2w64_c') }}         # [win]
+        - {{ compiler('m2w64_cxx') }}       # [win]
+        - {{ compiler('m2w64_fortran') }}   # [win]
+        - cmake
+        - ninja
+        - pkgconfig
 
-requirements:
-  build:
-    - {{ stdlib('c') }}                 # [unix]
-    - {{ compiler('c') }}               # [unix]
-    - {{ compiler('cxx') }}             # [unix]
-    - {{ compiler('fortran') }}         # [unix]
-    - {{ stdlib('m2w64_c') }}           # [win]
-    - {{ compiler('m2w64_c') }}         # [win]
-    - {{ compiler('m2w64_cxx') }}       # [win]
-    - {{ compiler('m2w64_fortran') }}   # [win]
-    - cmake
-    - ninja
-    - pkgconfig
+      host:
+        - libblas
+        - libcblas
+        - liblapack
+        - liblapacke
+        # - libscotch # no 64bit support
+        - libhwloc
+        - metis
 
-  host:
-    - libblas
-    - libcblas
-    - liblapack
-    - liblapacke
-    # - libscotch # no 64bit support
-    - libhwloc
-    - metis
-
-
-test:
-  commands:
-    # no idea how to test pastix...
+  - name: "{{ name }}_python"
+    build:
+      noarch: python
+    requirements:
+      run:
+        - {{ name }}
+        - python
+        - numpy
+    files:
+      include:
+        - {{ PREFIX }}/lib/python*/site-packages/pastix/*
+        - {{ PREFIX }}/lib/python*/site-packages/spm/*
```

Comment:
i do'nt think the noarch stuff really works like this does it? 

You might want to try to follow the ideas from:
https://github.com/conda-forge/zaber-motion-feedstock/blob/main/recipe/meta.yaml

Reply:
Thanks, I will try this

Reply:
Or maybe build for all different python versions, there are some python files created during the build process, not sure if noarch works.